xml/System/DateTime.xml

```diff
@@ -6825,9 +6825,13 @@ The following example demonstrates the <xref:System.DateTime.Parse%28System.Stri
 ## Remarks  
  A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond, or 10 million ticks in a second.  
   
- The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 (0:00:00 UTC on January 1, 0001, in the Gregorian calendar), which represents <xref:System.DateTime.MinValue?displayProperty=nameWithType>. It does not include the number of ticks that are attributable to leap seconds.  
-  
-   
+The value of this property represents the number of 100-nanosecond intervals that have elapsed since 12:00:00 midnight, January 1, 0001 in the Gregorian calendar, which represents DateTime.MinValue. It does not include the number of ticks that are attributable to leap seconds.
+If the DateTime object has Kind property set to Local, that means the ticks inside this object is representing the time elapsed time since 12:00:00 midnight, January 1, 0001 in the local time specified by the current time zone setting.
+If the DateTime object has Kind property set to Utc, that means the ticks inside this object is representing the time elapsed time since 12:00:00 midnight, January 1, 0001 in the Coordinated Universal Time.
```

Comment:
```suggestion
If the `DateTime` object has its `Kind` property set to `Utc`, its ticks represent the time elapsed time since 12:00:00 midnight, January 1, 0001 in the Coordinated Universal Time.
```

Reply:
@awilco this one was resolved but not applied. Can you apply it? I think we can merge this